src-tauri/src/northstar/install.rs

```diff
@@ -94,11 +94,29 @@ async fn do_install(
         .unwrap();
 
     log::info!("Extracting Northstar...");
-    extract(nfile, std::path::Path::new(&game_install.game_path))?;
+    extract(nfile, std::path::Path::new(&extract_directory))?;
+
+    log::info!("Installing Northstar...");
+
+    for entry in std::fs::read_dir(extract_directory).unwrap() {
+        let entry = entry.unwrap();
+        let destination = format!(
+            "{}/{}",
+            game_install.game_path,
+            entry.path().file_name().unwrap().to_str().unwrap()
+        );
+
+        log::info!("Installing {}", entry.path().display());
+        if !entry.file_type().unwrap().is_dir() {
+            std::fs::rename(entry.path(), destination)?;
+        } else {
+            move_dir_all(entry.path(), destination)?;
+        }
```

Comment:
Is there no generic OS move function that works for both files and folders?

Reply:
None that I found.
fs::rename can do some directory work on Linux, but its semantics make it unreliable.

Reply:
Aight, I feel like there should be an easier way then having our own code for doing it but if it works, I guess it's good enough for now (until the next refactor :trollface:)

Reply:
Directory related APIs tend to be very washy cross platform and standard rust provides nothing outside of creation and deletion.

the `fs_extra` crate provides what we want in a way similar to the stdlib.